torch/nn/quantized/modules/activation.py

```diff
@@ -31,7 +31,6 @@ class ReLU(torch.nn.ReLU):
     """
     def __init__(self, inplace=False):
         super(ReLU, self).__init__(inplace)
-        assert not inplace, 'torch.nn.quantized.ReLU does not support inplace'
 
     def forward(self, input):
         return torch.nn.quantized.functional.relu(input)
```

Comment:
should `inplace = self.inplace` be then passed to `torch.nn.quantized.functional.relu` call?

Reply:
Good point! Missed that!